### PR TITLE
Add barebones deploy scripts to make it work

### PR DESCRIPTION
**Pull request by azizova-leyla**
Issues: https://github.com/seattle-uat/civiform/issues/2553

### CODE REVIEW

File: cloud/aws/bin/deploy [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#! /usr/bin/env bash
+set -e
+
+source "cloud/shared/bin/lib.sh"
+source "cloud/azure/bin/lib.sh"
```
Comment (kmcnellis) [inline cloud/aws/bin/deploy:5]: Should this be importing anything from azure?

**Review by kmcnellis — CHANGES_REQUESTED** [review]

File: cloud/aws/bin/deploy (hunk 1 above)
Comment (azizova-leyla) [reply cloud/aws/bin/deploy:5]: good catch. we don't need this.

**Review by bion — APPROVED** [review]

File: cloud/aws/bin/deploy [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#! /usr/bin/env bash
+set -e
```
Comment (goldblatt) [inline cloud/aws/bin/deploy:2]: we could possibly make these scripts python based?

**Review by goldblatt — APPROVED** [review]
could possibly make this python based but otherwise seems alright

**Review by kmcnellis — APPROVED** [review]

**Pull request merged**